docs/authentication.md

```diff
@@ -1,14 +1,23 @@
 # Authentication
-Harvester Dashboard supports `local auth` mode for authentication. The default username and password is `admin/password`.
+
+>  Available as of v0.2.0
+
+With ISO installation mode, user will be prompted to set the root password for the default `admin` user on the first-time login.
+
+![auth](./assets/first-log-in.png)
+
 
 The Harvester login page is shown below:
 
 ![auth](./assets/authentication.png)
 
 
+
 ## App Mode
 In `App mode`, which is intended only for development and testing purposes, more authentication modes are configurable using the environment variable `HARVESTER_AUTHENTICATION_MODE`.
 
+Harvester Dashboard default to use `local auth` mode for authentication. The default username and password is `admin/password`.
+
 The currently supported options are `localUser` (the same as `local auth` mode) and `kubernetesCredentials`.
 
-If the `kubernetesCredentials` authentication option is used, either a kubconfig file or bearer token can provide access to Harvester.
\ No newline at end of file
+If the `kubernetesCredentials` authentication option is used, either a `KUBCONFIG` file or bearer token can provide access to Harvester.
```

Comment:
We don't need to capitalize it. Kubeconfig is a common word(https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). It was a typo missing e though.